tests/hub-local/test_hosted_instance_deletion_gate.py

```diff
@@ -27,13 +27,13 @@ def test_hosted_instance_deletion_gate(run_id, s3_bucket):
 
     # Make sure gating function blocks instance and storage deletion if
     # storage location is not empty
-    path.fs.touch(str(path / "new_file"))
+    path.touch(str(path / "new_file"))
     instance_slug = f"{settings.user.handle}/{test_instance.name}"
     with pytest.raises(ValueError):
         delete_instance(instance_slug)
 
     # Make sure instance and storage deletion is possible with empty storage
-    path.fs.rm(str(path / "new_file"))
+    path.unlink(str(path / "new_file"))
```

Comment:
This should read:

```
(path / "new_file").unlink()
```

But ideally you make a new path for the new file!

Paths are pretty objects that should be used 😍